src/Controls/samples/Controls.Sample/Startup.cs

```diff
@@ -36,6 +36,13 @@ enum PageType { Xaml, Semantics, Main, Blazor, NavigationPage, Shell }
 
 		public void Configure(IAppHostBuilder appBuilder)
 		{
+			bool useFullDIAndBlazor = UseFullDI
+
+#if BLAZOR_ENABLED
```

Comment:
I don't think this needs to be guarded because if you somehow pick Blazor outside of .NET 6, it'll throw later anyway.

Reply:
(That is, I don't think the `#if` is needed)